ch_tools/chadmin/cli/object_storage_group.py

```diff
@@ -172,7 +174,17 @@ def _clean_object_storage(
     Delete orphaned objects from object storage.
     """
     disk_conf: S3DiskConfiguration = ctx.obj["disk_configuration"]
-    prefix = object_name_prefix or disk_conf.prefix
+
+    if object_name_prefix is not None:
+        prefix = object_name_prefix
+    else:
+        prefix = disk_conf.prefix
+        # Yandex cloud specific
+        # Remove the last shard section from prefix
```

Comment:
Are you sure that make some specific behavior as default is a right way?
For example, in DC we name shards as `s#`, not `shard#`.

Reply:
Agree, this is a very questionable change. Removed it from this PR.